kratos/python_scripts/response_functions/response_function_interface.py

```diff
@@ -1,16 +1,5 @@
-
-# ==============================================================================
-#  KratosShapeOptimizationApplication
-#
-#  License:         BSD License
-#                   license: ShapeOptimizationApplication/license.txt
-#
-#  Main authors:    Geiser Armin, https://github.com/armingeiser
-#
-# ==============================================================================
-
-class ResponseFunctionBase(object):
-    """The base class for response functions. Each response function
+class ResponseFunctionInterface(object):
```

Comment:
The header is missing in this file.

Reply:
There are no headers in the core python files, that's why i removed it. 

By calling it interface I tried to avoid the confusion with the cpp base class for adjoint responses.